managed/utils/encryption/helpers.go

```diff
@@ -0,0 +1,177 @@
+	for _, t := range columns {
+		switch t {
+		case "VARCHAR", "JSONB":
+			row = append(row, &sql.NullString{})
+		default:
+			return nil, fmt.Errorf("unsupported identificator type %s", t)
+		}
+	}
+
+	return row, nil
+}
+
+func encryptColumnStringHandler(e *Encryption, val any) (any, error) {
+	value := val.(*sql.NullString) //nolint:forcetypeassert
+	if !value.Valid {
+		return sql.NullString{}, nil
+	}
+
+	encrypted, err := e.Encrypt(value.String)
+	if err != nil {
+		return nil, err
+	}
+
+	return encrypted, nil
+}
+
+func decryptColumnStringHandler(e *Encryption, val any) (any, error) {
+	value := val.(*sql.NullString) //nolint:forcetypeassert
+	if !value.Valid {
+		return nil, nil //nolint:nilnil
+	}
+
+	decrypted, err := e.Decrypt(value.String)
+	if err != nil {
+		return nil, err
+	}
+
+	return decrypted, nil
+}
+
+func (e *Encryption) getPrimitive() (tink.AEAD, error) { //nolint:ireturn
+	serializedKeyset, err := base64.StdEncoding.DecodeString(e.Key)
+	if err != nil {
+		return nil, err
+	}
+
+	binaryReader := keyset.NewBinaryReader(bytes.NewBuffer(serializedKeyset))
+	parsedHandle, err := insecurecleartextkeyset.Read(binaryReader)
+	if err != nil {
+		return nil, err
+	}
+
+	return aead.New(parsedHandle)
+}
+
+func (e *Encryption) generateKey() error {
+	handle, err := keyset.NewHandle(aead.AES256GCMKeyTemplate())
+	if err != nil {
+		return err
+	}
+
+	buff := &bytes.Buffer{}
+	err = insecurecleartextkeyset.Write(handle, keyset.NewBinaryWriter(buff))
+	if err != nil {
+		return err
+	}
+	e.Key = base64.StdEncoding.EncodeToString(buff.Bytes())
+
+	return e.saveKeyToFile()
+}
+
+func (e *Encryption) saveKeyToFile() error {
+	return os.WriteFile(e.Path, []byte(e.Key), 0o644) //nolint:gosec
+}
+
+func (table Table) columnsList() []string {
+	res := []string{}
+	for _, c := range table.Columns {
+		res = append(res, c.Name)
+	}
+
+	return res
+}
+
+func (table Table) read(tx *reform.TX) (*QueryValues, error) {
+	what := slices.Concat(table.Identificators, table.columnsList())
+	query := fmt.Sprintf("SELECT %s FROM %s", strings.Join(what, ", "), table.Name)
+	rows, err := tx.Query(query)
+	if err != nil {
+		return nil, err
+	}
+
+	q := &QueryValues{}
+	for rows.Next() {
+		row, err := prepareRowPointers(rows)
+		if err != nil {
+			return nil, err
+		}
+		err = rows.Scan(row...)
+		if err != nil {
+			return nil, err
+		}
+
+		i := 1
+		set := []string{}
+		setValues := []any{}
+		for k, v := range row[len(table.Identificators):] {
+			set = append(set, fmt.Sprintf("%s = $%d", table.Columns[k].Name, i))
+			setValues = append(setValues, v)
+			i++
+		}
+		setSQL := fmt.Sprintf("SET %s", strings.Join(set, ", "))
+		q.SetValues = append(q.SetValues, setValues)
+
+		where := []string{}
+		whereValues := []any{}
+		for k, id := range table.Identificators {
+			where = append(where, fmt.Sprintf("%s = $%d", id, i))
+			whereValues = append(whereValues, row[k])
+			i++
+		}
+		whereSQL := fmt.Sprintf("WHERE %s", strings.Join(where, " AND "))
```

Comment:
```suggestion
		whereSQL := "WHERE " + strings.Join(where, " AND ")
```

Reply:
There is a new lint rule (don't remember exactly which one), that will go red here claiming we can do the same without calling Sprintf.